app/src/main/java/fr/free/nrw/commons/di/CommonsApplicationModule.java

```diff
@@ -231,10 +234,55 @@ public QuadTree providesQuadTres() {
     @Provides
     @Singleton
     public AppDatabase provideAppDataBase() {
-        appDatabase=Room.databaseBuilder(applicationContext, AppDatabase.class, "commons_room.db").build();
+        Builder<AppDatabase> appDatabaseBuilder = Room
+            .databaseBuilder(applicationContext, AppDatabase.class, "commons_room.db");
+        appDatabaseBuilder.addMigrations(MIGRATION_1_2);
+        appDatabase = appDatabaseBuilder.build();
         return appDatabase;
     }
 
+    static final Migration MIGRATION_1_2 = new Migration(1, 2) {
+        @Override
+        public void migrate(SupportSQLiteDatabase database) {
+            database.execSQL("DROP TABLE `contribution`");
```

Comment:
If we just delete everything instead of migrating data we could `fallbackDestructively` like mentioned [here](https://developer.android.com/training/data-storage/room/migrating-db-versions#handle-missing-migrations)

Reply:
Thanks, @macgills. Will make this change.